### PR TITLE
Add commands to add and remove datastores

### PR DESCRIPTION
**Pull request by dougm**

### CODE REVIEW

File: govc/datastore/nas/create.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+func (cmd *create) Register(f *flag.FlagSet) {
+	f.StringVar(&cmd.AccessMode, "mode", string(types.HostMountModeReadWrite), "Access mode for the mount point")
+	f.StringVar(&cmd.LocalPath, "local-path", "", "Name of the NAS datastore")
+	f.StringVar(&cmd.RemoteHost, "remote-host", "", "Remote hostname of the NAS datastore")
+	f.StringVar(&cmd.RemotePath, "remote-path", "", "Remote path of the NFS mount point")
+	f.StringVar(&cmd.Type, "type", "", "Type of the the NAS volume")
+	f.StringVar(&cmd.UserName, "username", "", "Username to use when connecting (CIFS only)")
+	f.StringVar(&cmd.UserName, "password", "", "Password to use when connecting (CIFS only)")
```
Comment (pietern) [inline govc/datastore/nas/create.go]: Stores in the Username field ;)

File: govc/datastore/nas/create.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+func init() {
+	cli.Register("datastore.nas.create", &create{})
+}
+
+func (cmd *create) Register(f *flag.FlagSet) {
+	f.StringVar(&cmd.AccessMode, "mode", string(types.HostMountModeReadWrite), "Access mode for the mount point")
+	f.StringVar(&cmd.LocalPath, "local-path", "", "Name of the NAS datastore")
+	f.StringVar(&cmd.RemoteHost, "remote-host", "", "Remote hostname of the NAS datastore")
+	f.StringVar(&cmd.RemotePath, "remote-path", "", "Remote path of the NFS mount point")
+	f.StringVar(&cmd.Type, "type", "", "Type of the the NAS volume")
```
Comment (pietern) [inline govc/datastore/nas/create.go]: What kind of values can be used here?

**Comment by dougm** [comment]
Thanks @pietern , fixed those and also added a -force flag.

**Comment by pietern** [comment]
Great, looks good :+1:

**Pull request merged**